Makefile.am

```diff
@@ -12,6 +12,7 @@ sstconf_DATA = src/sst/sstsimulator.conf
 EXTRA_DIST = \
 	autogen.sh \
 	CONTRIBUTING.md \
+	scrips/clang-format-test.sh \
```

Comment:
Should this be `scripts` not `scrips` ?

Reply:
🤷‍♀️ yeah probably

Reply:
Have you developed love or hatred for the auto-tester yet ? :)

Reply:
I just try to ignore it :)